src/families/tron/types.js

```diff
@@ -13,22 +13,118 @@ export type CoreOperationSpecifics = {};
 
 export type CoreCurrencySpecifics = {};
 
+export type TronOperationMode = "send" | "freeze" | "unfreeze" | "vote" | "claimReward";
+export type TronResource = "BANDWIDTH" | "ENERGY";
+
 export type NetworkInfo = {|
-  family: "tron"
+  family: "tron",
+  freeNetUsed: number,
+  freeNetLimit: number,
+  NetUsed: number,
+  NetLimit: number,
+  EnergyLimit: ?number
 |};
 
 export type NetworkInfoRaw = {|
-  family: "tron"
+  family: "tron",
+  freeNetUsed: number,
+  freeNetLimit: number,
+  NetUsed: number,
+  NetLimit: number,
+  EnergyLimit: ?number
 |};
 
 export type Transaction = {|
   ...TransactionCommon,
-  family: "tron"
+  family: "tron",
+  mode: TronOperationMode,
+  resource: ?TronResource,
+  networkInfo: ?NetworkInfo,
+  duration: ?number,
+  votes: Vote[]
 |};
 
 export type TransactionRaw = {|
   ...TransactionCommonRaw,
-  family: "tron"
+  mode: TronOperationMode,
+  family: "tron",
+  resource: ?TronResource,
```

Comment:
i'm not sure to understand why the "resource" itself belongs in Transaction if it's already in context of Account?